### PR TITLE
doc: fix powershell redir

### PR DESCRIPTION
**Pull request by janlazo**
Tested using with my vimrc, https://github.com/junegunn/vim-plug/pull/860, on Vim and Neovim.

### CODE REVIEW

File: runtime/doc/options.txt [hunk 1]
```diff
@@ -5159,8 +5159,9 @@ A jump table for the options with a short description can be found at |Q_op|.
 	unescaping, so to keep yourself sane use |:let-&| like shown above.
 							*shell-powershell*
 	To use powershell (on Windows): >
-		set shell=powershell shellquote=( shellpipe=\| shellredir=> shellxquote=
+		set shell=powershell shellquote=( shellpipe=\| shellxquote=
 		set shellcmdflag=-NoLogo\ -NoProfile\ -ExecutionPolicy\ RemoteSigned\ -Command
+		set shellredir=\|\ Out-File\ -Encoding\ ASCII
```
Comment (justinmk) [inline runtime/doc/options.txt]: would this be a problem for utf-8?

File: runtime/doc/options.txt [hunk 2]
```diff
@@ -5159,8 +5159,9 @@ A jump table for the options with a short description can be found at |Q_op|.
 	unescaping, so to keep yourself sane use |:let-&| like shown above.
 							*shell-powershell*
 	To use powershell (on Windows): >
-		set shell=powershell shellquote=( shellpipe=\| shellredir=> shellxquote=
+		set shell=powershell shellquote=( shellpipe=\| shellxquote=
```
Comment (justinmk) [inline runtime/doc/options.txt]: I wonder what `>`  is equivalent to exactly.

File: runtime/doc/options.txt (hunk 1 above)
Comment (janlazo) [reply runtime/doc/options.txt]: https://www.w3.org/2001/06/utf-8-test/UTF-8-demo.html
`ASCII` encoding corrupts the file.

```ps1
Get-Content -Encoding UTF8 UTF-8-demo.html | Out-File foo.txt
```
foo.txt seems to retain the utf-8 characters if omitting `-Encoding` or using `utf8`, `utf32`, `unicode` encoding values.

**Comment by blueyed** [comment]
Should this be done here then also (at least)?
https://github.com/blueyed/neovim/blob/374a35b044ce324076bd7639dd4db08952f245b3/test/functional/helpers.lua#L498-L503

**Comment by janlazo** [comment]
Sure. Neovim can handle `set shellredir=>` but my shellredir suggestion should work for the functional tests.

File: runtime/doc/options.txt (hunk 2 above)
Comment (janlazo) [reply runtime/doc/options.txt:5162]: utf-16 on Windows.
utf-8 on Unix.

**Pull request merged**